### PR TITLE
Sync all nodes

### PR DESCRIPTION
**Pull request by Istani**
I guess that it would be great to be able to sync all known nodes at once to a new guild member or player.  So i implemented a "/gather sync" command.
But i am not sure if my remote Call would break something... But should be okay...

### CODE REVIEW

**Comment by Istani** [comment]
Oh Sorry is just see there is already an issue for that...
https://github.com/Wuild/GatherLite/issues/29

File: methods.lua [hunk 1]
```diff
@@ -699,6 +704,62 @@ GatherLite.p2p = function()
...
+        else
+            for k, node in ipairs(GatherLiteGlobalSettings.database[skill]) do
+                table.insert(database, node);
+            end
+        end
+        last_skill=skill;
+    end
+    last_skill = nil;
+
+    for i = 5, 1, -1 do
```
Comment (MarkusTeufelberger) [inline methods.lua:733]: Probably this function should not just send 5 random nodes...?

You'd need rate limiting though, sending more than a few hundred characters of message per second will get you kicked from the servers.

**Review by Wuild — COMMENTED** [review]
I don’t really understand what your trying todo here.

**Comment by Istani** [comment]
Oh Sorry i am new to this and was trying to help with a 
/gather sync
Command... But yeah I should learn more Basics of writing Addons (MarkusTeufelberger just pointed to something i didnt know)

**Comment by Wuild** [comment]
I don’t mind help. I just don’t understand what the code does.

**Comment by Istani** [comment]
Well now that i look again over it...
Its your p2p thing a bit diffrent XD
Yeah Nonsens^^ Should have take a few more coffee...

File: methods.lua (hunk 1 above)
Comment (Istani) [reply methods.lua:733]: Yeah Nonsens^^ Should have take a few more coffee...

Thank you for pointing that out

Comment (Istani) [reply methods.lua:733]: mhm maybe adding something like a que to sending it...

**Pull request closed without merging**